x-pack/test/security_solution_cypress/cypress/tasks/create_new_rule.ts

```diff
@@ -763,6 +774,28 @@ export const selectAndLoadSavedQuery = (queryName: string, queryValue: string) =
   cy.get(CUSTOM_QUERY_INPUT).should('have.value', queryValue);
 };
 
+export const enablesAndPopulatesThresholdSuppression = (
+  interval: number,
+  timeUnit: 's' | 'm' | 'h'
+) => {
+  // enable suppression is unchecked so the rest of suppression components are disabled
+  cy.get(ALERT_SUPPRESSION_DURATION_PER_TIME_INTERVAL).should('be.disabled').should('be.checked');
+  cy.get(ALERT_SUPPRESSION_DURATION_PER_RULE_EXECUTION)
+    .should('be.disabled')
+    .should('not.be.checked');
+
+  // enables suppression for threshold rule
+  // checkbox input covered by label element, so force:true is needed
+  cy.get(THRESHOLD_ENABLE_SUPPRESSION_CHECKBOX).should('not.be.checked').check({ force: true });
```

Comment:
Same here for the force, it is extremely necessary?

Reply:
force removed